Test/Version.Tests.ps1

```diff
@@ -401,7 +401,7 @@ Describe 'Version' {
 
     It 'should use prerelease version from the PowerShell Gallery' {
         GivenFile 'Whiskey.psd1' '@{ ModuleVersion = ''0.41.1'' ; PrivateData = @{ PSData = @{ Prerelease = ''beta.1'' } } }'
-        GivenProperty @{ Path = 'Whiskey.psd1' }
+        GivenProperty @{ Path = 'Whiskey.psd1'; IncrementPrereleaseVersion = $true; }

```

Comment:
Are there any existing tests that assert the behavior for when this is `$false` ?